.github/actions/bootstrap/action.yml

```diff
@@ -0,0 +1,29 @@
+name: "bootstrap"
+description: "Bootstrap Python environment including Python dependencies and configure optimizations."
+inputs:
+  python-version:
+    description: "Version range or exact version of Python or PyPy to use, using SemVer's version range syntax."
```

Comment:
Why PyPy?

Reply:
@danieljanes this is just passed through to the office action and the description is a copy.